app/main.py

```diff
@@ -2,0 +1,17 @@
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    mv, source_file, destination_file = split_command
```

Comment:
Code Style Issue: The variable names `mv`, `source_file`, and `destination_file` are not very descriptive. Consider renaming them to something more meaningful, like `command_name`, `source_path`, and `destination_path`.